src/main/java/org/apache/bcel/generic/Type.java

```diff
@@ -17,7 +17,6 @@
  */
 package org.apache.bcel.generic;
 
-
```

Comment:
This is a good change to bring typetools/commons-bcel in accordance with its upstream (thanks), but it isn't a change from the pull request.  It would have been clearer to fix gratuitous differences from upstream first, in a separate commit.  (But, once irrelevant changes like this are eliminated, they should never come back.)

The same comment applies to some other changes, like those in .gitignore.